packages/account/src/providers/operations.graphql

```diff
@@ -686,15 +686,43 @@ query estimatePredicates($encodedTransaction: HexString!) {
 
 query getBlock($blockId: BlockId, $height: U32) {
   block(id: $blockId, height: $height) {
-    ...blockFragment
+    height
```

Comment:
Why not change the `blockFragment` directly?

Reply:
This is the `blockFragment` currently:
```gql
fragment blockFragment on Block {
  id
  height
  header {
    time
    daHeight
    stateTransitionBytecodeVersion
    transactionsCount
    transactionsRoot
    messageOutboxRoot
    eventInboxRoot
    prevRoot
    applicationHash
  }
  transactions {
    id
  }
}
```
I can't find a change between this fragment and what you did with the fields.

Reply:
Oh that's a bummer. I thought `blockFragment` meant us relaying the whole object without extracting anything specific. My bad. Marking as draft for now

Reply:
I overlooked the fact that `blockFragment` must have been implemented by _us_ somewhere 🙃